client/backoff/backoff_test.go

```diff
@@ -92,8 +92,8 @@ func TestJitter(t *testing.T) {
 
 func TestRetry(t *testing.T) {
 	b := Backoff{
-		Min:    time.Duration(1),
-		Max:    time.Duration(100),
+		Min:    time.Duration(50) * time.Millisecond,
+		Max:    time.Duration(200) * time.Millisecond,
```

Comment:
These can just be like `Min: 50 * time.Millisecond` can't they? 

(Looks like there's lots of unnecessary casts in this file)